index.html

```diff
@@ -0,0 +1,370 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="mobile.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="setup">
+      <header>
+        <a href="">My Log</a>
+        <nav>
+          <img
+            src="./images/Vector.png"
+            alt="image was moved to another location"
+          />
+        </nav>
+      </header>
+      <section>
+        <div class="bg">
+          <img id="Ellipse11" src="./images/Ellipse11.png" alt="ellipse11" />
+          <img id="Ellipse3" src="images/Ellipse3.png" alt="ellipse3" />
+          <img id="Rectangle4" src="./images/Rectangle4.png" alt="rectangle4" />
+        </div>
+        <h1>I`m Joffrey NKESHIMANA, Glad to see you!</h1>
```

Comment:
- [ ] A friendly reminder that you should only use and h1 tag once across the markup of a page, kindly update all other h1 tags to an hx tag depending on their priority and structure.